### ISSUE TITLE
Accessibility for removable chips

### ISSUE THREAD

**Issue by tinayuangao** [angular/material2#5925]
#### Bug, feature request, or proposal:
feature

#### What is the expected behavior?
User should be able to use keyboard to remove a chip.

#### What is the current behavior?
User cannot user keyboard to remove a chip

#### What are the steps to reproduce?

Providing a Plunker (or similar) is the *best* way to get the team to see your issue.
Plunker template: https://goo.gl/DlHd6U

**Comment by tinayuangao** [comment]
Delete key is used to remove a chip. Closing this issue
